weave/trace_server/costs/create_new_migration.py

```diff
@@ -0,0 +1,255 @@
+# This script creates a new migration for the llm_token_prices table
```

Comment:
this is a script we would run periodically to generate a new migration for the cost table, or we can create manual migrations